app/models/student_risk_level.rb

```diff
@@ -55,7 +55,10 @@ def update_risk_level!
   def calculate_level
     # As defined by Somerville Public Schools
 
-    if mcas_or_star_at_level(3) || limited_english_proficiency_risk_level == 3
+    if student.school.school_type == "HS"
+      #High School risk levels have not been defined
+      level = nil
+    elsif mcas_or_star_at_level(3) || limited_english_proficiency_risk_level == 3
```

Comment:
This effectively removes risk levels from high school students. We haven't determined the logic but it will be different than at the elementary school level.